_i18n/ar/resources/moneropedia/merge-mining.md

```diff
@@ -11,10 +11,4 @@ summary: "the process of mining two or more blockchains at the same time "
 Merge mining is the process of @mining two or more @blockchains at the same time. Generally there is a parent @blockchain and an auxillary blockchain both of which use the same proof of work algorithm. In most cases the auxillary chain will have a lower difficulty than the parent block chain.
 
 Merge mining can be used for securing a lower hashrate blockchain or developing a @sidechain that does something different than the primary chain.
-Some examples that currently exist or are in development are: a blockchain game (TownForge), digital assets platform (Tari) or a decentralized mining pool (@P2Pool).
-
-#### External Links
-
-- [Tari - What is Tari](https://www.tari.com/#what-is-tari)
-- [Townforge -About](https://townforge.net/about)
-- [P2Pool GitHub](https://townforge.net/about)
+Some examples that currently exist or are in development are: a blockchain game [TownForge](https://townforge.net/about), digital assets platform [Tari](https://www.tari.com/#what-is-tari) or a decentralized mining pool (@P2Pool).
```

Comment:
`, and a decentralized mining pool`

Reply:
Ack.

Reply:
Frankly it looks cleaner with the bulleted list of links. Only problem is the P2Pool GitHub link is a copy/paste of the townforge link.